settings.gradle

```diff
@@ -12,7 +12,6 @@ pluginManagement {
 
 plugins {
     id 'com.gradle.develocity' version '3.17.2'
-    id 'org.gradle.toolchains.foojay-resolver-convention' version '0.8.0'
```

Comment:
This might be necessary for installing toolchain.

Reply:
And I removed it because I disabled toolchain to allow building with recent Java versions while targeting older bytecode.